src/bun.js/bindings/bindings.zig

```diff
@@ -4241,6 +4241,13 @@ pub const JSValue = enum(JSValueReprInt) {
         return JSBuffer__isBuffer(global, value);
     }
 
+    pub fn isArrayBufferView(val: JSC.JSValue, globalThis: *JSC.JSGlobalObject) bool {
```

Comment:
Is this method necesary?

`.asArrayBuffer()` returns null if the value is not an ArrayBufferView.